pkg/kritis/crd/securitypolicy/securitypolicy.go

```diff
@@ -69,7 +70,7 @@ func ValidateImageSecurityPolicy(isp v1beta1.ImageSecurityPolicy, image string,
 		return violations, nil
 	}
 	// Now, check vulnz in the image
-	vulnz, err := client.Vulnerabilities(image)
+	vulnz, err := client.Vulnerabilities(image, auths)
```

Comment:
This actually does not need auths.  Vulnerability Occurrences reside in the same project as the image, so this should stay as is.